packages/ui/.eslintrc

```diff
@@ -35,21 +35,16 @@
         "prefer-destructuring": 0,
         "no-underscore-dangle": 0,
         "no-shadow": 0,
+        "import/no-anonymous-default-export": 0,
         "no-unused-vars": [
             1,
             {
                 "ignoreRestSiblings": false
             }
         ],
         "prettier/prettier": [
-            2,
+            "error",
```

Comment:
what's this `error` for?

Reply:
it is an alias, 

0 : off
1: warning
2: error

I use English word to make it readable

Reply:
soon will change every number to english word

Reply:
basically what it does is, linter will raise error if any of prettier rule is violated,